### DISCUSSION TITLE
System monitor

### DISCUSSION THREAD

**Discussion by JacksonJS32Kelly** [elementary/triage#393] [Q&A]
The elementary OS lacks programs for system monitoring of resources and processes

**Comment by ealaney** [comment]
The primary focus for elementary OS, as I understand it, is to provide a clean, minimalist environment for everyday use. While many people find a system monitor useful, many others do not, so it is an example of a program that is left to the user to select and install from AppCenter or Flathub.

Could one be included out of the box? Sure. But if that were done for each utility that gets used by a few people but not the majority, pretty soon elementary OS would be as big and bloated as Windows or MacOS.

**Comment by JacksonJS32Kelly** [comment]
Sorry but i doesn't agree with you... System monitor is not a bloat its
about monitoring your resources... Bloating on windows or mac its crippy
telemetry. But i agreed i can install this from gnome)

сб, 6 січ. 2024 р., 23:56 користувач Eric Laney ***@***.***>
пише:
